scripts/utils/migrate/fetch-docs.js

```diff
@@ -1,30 +1,37 @@
 const path = require('path');
 const docsApi = require('./docs-api');
 const { prop } = require('../functional');
-const { TYPES, ITEMS_PER_TYPE, DIRECT_IDS } = require('../constants');
+const {
+  TYPES,
+  ITEMS_PER_TYPE,
+  DIRECT_IDS,
+  MAX_PAGES_PER_TYPE,
+} = require('../constants');
 
 const IGNORED_TYPES = [TYPES.ATTRIBUTE_DEFINITION, TYPES.EVENT_DEFINITION];
 
 const fetchDocs = async () => {
   const requests = Object.values(TYPES)
     .filter((type) => !IGNORED_TYPES.includes(type))
     .map((type) =>
-      docsApi.get(path.join('/api/migration/content', type, 'list'), {
+      docsApi.paginate(path.join('/api/migration/content', type, 'list'), {
         perPage: ITEMS_PER_TYPE[type],
+        maxPages: MAX_PAGES_PER_TYPE[type] || Infinity,
       })
-    );
+    )
+    .concat(docsApi.paginate('/api/migration/content/page/nr-only/list'));
 
   const hardCodedRequests = DIRECT_IDS.map((id) =>
     docsApi.get(`/api/migration/content/page/${id}`)
   );
 
-  const nrOnly = await docsApi.paginate(
-    '/api/migration/content/page/nr-only/list'
-  );
-
-  const docs = await Promise.all([...requests, ...hardCodedRequests]);
+  const docs = await Promise.all(requests);
+  const hardCodedDocs = await Promise.all(hardCodedRequests);
```

Comment:
I think I missed this previously, what are these NR hardcoded docs?

Reply:
We were asked to migrate a subset of integration docs so that Fred's team could take a look at them. This ensures that we get back those integration pages.